CalibTracker/SiPixelESProducers/interface/alpaka/SiPixelGainCalibrationForHLTUtilities.h

```diff
@@ -0,0 +1,48 @@
+
+#ifndef CalibTracker_SiPixelESProducers_interface_alpaka_SiPixelGainUtilities_h
+#define CalibTracker_SiPixelESProducers_interface_alpaka_SiPixelGainUtilities_h
+
+#include <cstdint>
+#include <alpaka/alpaka.hpp>
+#include "DataFormats/Portable/interface/alpaka/PortableCollection.h"
+#include "DataFormats/SiPixelGainCalibrationForHLTSoA/interface/SiPixelGainCalibrationForHLTLayout.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  struct SiPixelGainUtilities {
+    ALPAKA_FN_HOST_ACC ALPAKA_FN_ACC ALPAKA_FN_INLINE static std::pair<float, float> getPedAndGain(
+        uint32_t moduleInd,
+        int col,
+        int row,
+        bool& isDeadColumn,
+        bool& isNoisyColumn,
+        const SiPixelGainCalibrationForHLTSoAConstView& view) {
```

Comment:
This function looks like in the AoS model it would be a member function of the struct. I suggest we treat these kind of functions for SoAs such that
- they are defined in the same namespace (and file) as the Layout and View are defined (i.e. where the `GENERATE_SOA_LAYOUT()` macro is called)
- the `View` object is the first parameter (mimicking the implicit `this` argument of actual member functions)

The function call would then look like
```cpp
getPedAndGain(view, ...);
```
which would not be very different from
```cpp
view.getPedAndGain(...);
```

(by the way, I don't see anything in this file that would require the use of `ALPAKA_ACCELERATOR_NAMESPACE`)